packages/db/src/meta/pouch/adapters/types.ts

```diff
@@ -57,7 +57,7 @@ export namespace Generic {
     A extends Adapters,
     N extends AdapterName<A> = AdapterName<A>
   > {
-    workingDirectory?: string;
+    directory?: string;
```

Comment:
Besides removing this field here **and** in `ConnectOptions`, we could remove it from `ConnectOptions` but have `AttachOptions` require it;  we could have `Pouch.Adapters.concretize` do the the default data directory look up and pass that to `getDefaultAdapters`.

Don't have a strong opinion on which way is better, but it seems like low harm and less code just to have each adapter be responsible for the lookup... so I think I slightly prefer that? Curious your thoughts if you have any.